test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -118,95 +118,74 @@ class OverrideHostTest : public XdsEnd2endTest {
     return listener;
   }
 
-  std::multimap<std::string, std::string> SendRpcGetServerMetadata(
-      const grpc_core::DebugLocation& debug_location,
-      const RpcOptions& rpc_options) {
-    std::multimap<std::string, std::string> server_initial_metadata;
-    EchoResponse response;
-    grpc::Status status =
-        SendRpc(rpc_options, &response, &server_initial_metadata);
-    EXPECT_TRUE(status.ok())
-        << "code=" << status.error_code()
-        << ", message=" << status.error_message() << "\n"
-        << debug_location.file() << ":" << debug_location.line();
-    EXPECT_EQ(response.message(), kRequestMessage)
-        << debug_location.file() << ":" << debug_location.line();
-    return server_initial_metadata;
-  }
-
-  ClusterLoadAssignment EdsResourceWithChannelHealth(
-      absl::Span<const HealthStatus> backend_health_statuses) {
-    std::vector<EdsResourceArgs::Endpoint> endpoints;
-    int ind = 0;
-    for (HealthStatus status : backend_health_statuses) {
-      endpoints.emplace_back(CreateEndpoint(ind++, status));
+  absl::optional<std::vector<std::pair<std::string, std::string>>>
```

Comment:
It doesn't seem beneficial to use `absl::optional<>` here, since we can just return an empty vector if there's a problem.  In general, we only need `absl::optional<>` in cases where the contained type does not inherently have a value that we can use to indicate that it's not valid.